src/main/java/net/sf/jabref/logic/l10n/LocalizationKey.java

```diff
@@ -11,16 +11,18 @@ public LocalizationKey(String key) {
     }
 
     public String getPropertiesKeyUnescaped() {
-        // space, = and : are not allowed in properties file keys
+        // space, #, !, = and : are not allowed in properties file keys
```

Comment:
Why are'nt they repleaced here? - I don't get how the comment matches with the code.